src/reducers/entities/teams.ts

```diff
@@ -349,15 +353,15 @@ function groupsAssociatedToTeam(state = {}, action) {
         for (const group of groups) {
             associatedGroupIDs.delete(group.id);
         }
-        nextState[teamID] = Array.from(associatedGroupIDs);
+        nextState[teamID] = {ids: Array.from(associatedGroupIDs), totalCount: associatedGroupIDs.size};
```

Comment:
This looks like a bug returning the state, @mkraft what do you think?

Reply:
Yes it looks like a bug to me too just not reflected in the UX because of the way it's being used.